internal/sdkv2provider/resource_cloudflare_list_test.go

```diff
@@ -38,6 +38,7 @@ func TestAccCloudflareList_Exists(t *testing.T) {
 					resource.TestCheckResourceAttr(
 						name, "name", rnd),
 				),
+				Destroy: false,
```

Comment:
any reason why we are setting this here? the test should be excercising deletion as a part of the lifecycle flow.